playbooks/cloud-provider-openstack-acceptance-test-csi-manila/run.yaml

```diff
@@ -0,0 +1,208 @@
+- hosts: all
+  roles:
+    - config-golang
+    - export-cloud-openrc
```

Comment:
no need for this role if you don't use cloud service.

Reply:
ack